utils/utils.go

```diff
@@ -311,3 +314,47 @@ func GetSettingBool(settings map[string]interface{}, key string) bool {
 	}
 	return false
 }
+
+func readInput(ctx context.Context, reader *bufio.Reader, text, def string) (string, error) {
```

Comment:
i am suggesting moving these two funcs to the utils...to me this place looks more logical. i will keep old ones for now.
also, moving these over here would mean updating the signature, that has to accept `ctx` as a separate arg